src/lib/utils/data-handlers/resources/resource.ts

```diff
@@ -1,32 +1,41 @@
 import { env } from '$env/dynamic/public';
 import { fetchFromCacheOrCdn } from '$lib/data-cache';
 import { log } from '$lib/logger';
-import type { PassageResourceContent } from '$lib/types/passage';
-import { MediaType, type ResourceContentMetadata, type ResourceContentTiptap } from '$lib/types/resource';
+import type { FileManagerResourceContentInfo } from '$lib/types/file-manager';
+import {
+    MediaType,
+    type ResourceContentInfo,
+    type ResourceContentMetadata,
+    type ResourceContentTiptap,
+} from '$lib/types/resource';
 import { audioFileTypeForBrowser } from '$lib/utils/browser';
 
-export function resourceContentApiPath(resourceContent: PassageResourceContent) {
-    if (resourceContent.mediaTypeName === MediaType.Audio) {
-        return `resources/${resourceContent.contentId}/content?audioType=${audioFileTypeForBrowser()}`;
+export function resourceContentApiPath(resourceContent: ResourceContentInfo | FileManagerResourceContentInfo) {
```

Comment:
These functions now take two different types. One type is what the file manager will be using and one type is what the viewing page will be using.